c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -157,6 +157,24 @@ class TypeEncoder final : public TypeVisitor<TypeEncoder> {
           astEncoder(ast) {}
 
     void VisitQualType(const QualType &QT) {
+        if(isa<AtomicType>(QT)) {
```

Comment:
nit: looks like this could be `QT.isAtomicType()`.

Reply:
Nope -- gives `error: ‘const class clang::QualType’ has no member named ‘isAtomicType’`. (When applied only here, and also for other locations which was what I first tried as TBH the `isa<AtomicType>` is something I plainly copy-pasted from other locations in this C++ file).